public/js/components/Header/Header.js

```diff
@@ -39,6 +39,13 @@ class Header extends React.Component {
     this.props.atomActions.takeDownAtom(this.props.atom);
   }
 
+  isEditor = () => {
+    return location.pathname.search(new RegExp('\/atoms\/.*\/.*\/edit', 'g')) >= 0;
```

Comment:
I don't think we can do this as Header is a child of Page, which just has the route `/` - so the id and atomType parameters aren't available?